CairoMakie/src/primitives.jl

```diff
@@ -320,7 +320,7 @@ function draw_atomic(scene::Scene, screen::Screen, @nospecialize(primitive::Scat
     isempty(positions) && return
     size_model = transform_marker ? model : Mat4d(I)
 
-    font = to_font(to_value(get(primitive, :font, Makie.defaultfont())))
+    font = to_font(primitive.fonts, primitive.font[])
```

Comment:
fonts end up as `::Attributes` so I've just passed them directly. Not sure if that's correct though